website/src/site/styles/components/table.scss

```diff
@@ -34,7 +34,8 @@
       background-color: var(--table-code-background);
       border-radius: 0.6rem;
       box-decoration-break: clone;
-      box-decoration-break: clone;
+      /* stylelint-disable-next-line property-no-vendor-prefix */
+      -webkit-box-decoration-break: clone;
```

Comment:
I think you should have the vendor prefixes before the regular properties? So:

```css
-webkit-box-decoration-break: clone;
box-decoration-break: clone;
```

So it will will always use the non-prefixed one when available.